src/components/DatePicker/index.js

```diff
@@ -1,91 +1,82 @@
-import React from 'react';
 import moment from 'moment';
+import React, {useEffect, useRef} from 'react';
 import _ from 'underscore';
-import TextInput from '../TextInput';
 import CONST from '../../CONST';
 import * as Browser from '../../libs/Browser';
-import {propTypes, defaultProps} from './datepickerPropTypes';
+import TextInput from '../TextInput';
 import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
+import {defaultProps, propTypes} from './datepickerPropTypes';
 import './styles.css';
 
 const datePickerPropTypes = {
     ...propTypes,
     ...windowDimensionsPropTypes,
 };
 
-class DatePicker extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.setDate = this.setDate.bind(this);
-        this.showDatepicker = this.showDatepicker.bind(this);
+function DatePicker(props) {
```

Comment:
@srikarparsi 

[ContributingGuides](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring) suggests to use destructuring to get prop values.

> Always use destructuring to get prop values. Destructuring is necessary to assign default values to props.